PWGHF/TableProducer/HFTrackIndexSkimsCreator.cxx

```diff
@@ -117,9 +118,10 @@ struct HfTagSelCollisions {
     labels[2 + EventRejection::NContrib] = "rej. # of contributors";
     labels[2 + EventRejection::Chi2] = "rej. #it{#chi}^{2}";
     AxisSpec axisEvents = {nBinsEvents, 0.5, nBinsEvents + 0.5, ""};
-    registry.add("hEvents", "Events;;entries", HistType::kTH1F, {axisEvents});
+    histos[0] = std::get<std::shared_ptr<TH1>>(registry.add("hEvents", "Events;;entries", HistType::kTH1F, {axisEvents}));
```

Comment:
you can just do `registry.add<TH1>(...)`

Reply:
Thanks for the suggestion!

Reply:
It's likely that the overhead is small as you say, however I am trying to improve this task as much as possible since it often gives problems of CPU time. Let's see also what @vkucera and @ginnocen thinks about this.